docs/source/user-guide/lazy/execution.md

```diff
@@ -53,27 +53,18 @@ We look at [streaming in more detail here](streaming.md).
 
 While you're writing, optimizing or checking your query on a large dataset, querying all available data may lead to a slow development process.
 
-You can instead execute the query with the `.fetch` method. The `.fetch` method takes a parameter `n_rows` and tries to 'fetch' that number of rows at the data source. The number of rows cannot be guaranteed, however, as the lazy API does not count how many rows there are at each stage of the query.
+Instead, you can scan a subset of your partitions or use `.head`/`.collect` at the beginning and end of your query, respectively.
+Keep in mind that the results of aggregations and filters on subsets of your data may not be representative of the result you would get on the full data.
```

Comment:
`head`s that are after an aggregation don't influence an aggregation. I don't think this sentence makes much sense.

Reply:
The suggestion is to have `head` at the beginning of your query so in front of the aggregations.

Reply:
E.g.
```
q9 = (
    pl.scan_csv(f"docs/assets/data/reddit.csv")
    .head(10)
    .with_columns(pl.col("name").str.to_uppercase())
    .filter(pl.col("comment_karma") > 0)
    .fetch(n_rows=int(100))
    .collect()
)
```

Reply:
Is it too obvious? Maybe a little, but I have had these types of operations fail in ways I didn't expect and it is just single sentence so I feel like it is justified.

Reply:
Alright, with the accompanied query it makes sense. :+1: